websockets/speedups.c

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2009 Facebook
```

Comment:
Tornado is licensed under [Apache 2.0](https://github.com/tornadoweb/tornado/blob/master/LICENSE) license.

If you are ok with it, I'll address the issues raised in review.